### PR TITLE
Accept and reject curator from bounty list

### PR DESCRIPTION
**Pull request by ekowalsk**
image

### CODE REVIEW

File: packages/page-bounties/src/BountyCuratorProposedActions.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+}
+
+function BountyCuratorProposedActions ({ curatorId, index }: Props) {
+  const { t } = useTranslation();
+  const { api } = useApi();
+  const { allAccounts, hasAccounts } = useAccounts();
+
+  const isCurator = useMemo(() => allAccounts.includes(curatorId.toString()), [allAccounts, curatorId]);
+
+  return hasAccounts && isCurator
```
Comment (jacogr) [inline packages/page-bounties/src/BountyCuratorProposedActions.tsx:25]:
```suggestion
  return isCurator
```

Should be enough.

File: packages/page-bounties/src/BountyCuratorProposedActions.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+interface Props {
+  curatorId: AccountId;
+  index: number;
+}
+
+function BountyCuratorProposedActions ({ curatorId, index }: Props) {
+  const { t } = useTranslation();
+  const { api } = useApi();
+  const { allAccounts, hasAccounts } = useAccounts();
```
Comment (jacogr) [inline packages/page-bounties/src/BountyCuratorProposedActions.tsx:21]:
```suggestion
  const { allAccounts } = useAccounts();
```

... along with `isCurator` change

Comment (ekowalsk) [reply packages/page-bounties/src/BountyCuratorProposedActions.tsx]: Sorry, I did the force push :(

**Review by jacogr — APPROVED** [review]
Thank you.

**Pull request merged**